ansible/configs/ocp4-workshop/lifecycle.yml

```diff
@@ -91,7 +92,7 @@
           template:
             dest: "{{ output_dir }}/status.txt"
             src: files/status.j2
```

Comment:
You are missing an instance-state-name: stopped here - it should be in the original fork code.